services/motion/builtin/move_request.go

```diff
@@ -18,11 +18,18 @@ import (
 	"go.viam.com/rdk/spatialmath"
 )
 
+const (
+	defaultReplanCostFactor = 1.0
+	defaultMaxReplans       = 10
```

Comment:
The scope did not include an max number of replans. Instead, motion should continue trying indefinitely.

Reply:
I have a suspicion we may want this feature in the not too distant future.
Would you be okay with me leaving it in as-functioning, but setting the default to `math.Inf(1)`?

Reply:
yes, sounds good

Reply:
I don't see the reason to add this code if there's no way to modify it.  It will be easy to add back in later if we decide that we need it.